code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1329,7 +1329,7 @@ GLOBAL_LIST_EMPTY(features_by_species)
 	return TRUE
 
 /datum/species/proc/apply_damage(damage, damagetype = BRUTE, def_zone = null, blocked, mob/living/carbon/human/H, forced = FALSE, spread_damage = FALSE, wound_bonus = 0, bare_wound_bonus = 0, sharpness = NONE, attack_direction = null)
-	SEND_SIGNAL(H, COMSIG_MOB_APPLY_DAMAGE, damage, damagetype, def_zone, wound_bonus, bare_wound_bonus, sharpness, attack_direction)
+	SEND_SIGNAL(H, COMSIG_MOB_APPLY_DAMAGE, damage, damagetype, def_zone, blocked, wound_bonus, bare_wound_bonus, sharpness, attack_direction)
```

Comment:
Probably need to update the comment on this?

Reply:
The comment on MOB_APPLY_DAMAGE that specifies its argumnets

Reply:
Also are you sure that this doesn't break any existing consumers

Reply:
I checked every single use case, none of them used arguments beyond def_zone

Reply:
I'll update the comment soon